.env.example

```diff
@@ -0,0 +1 @@
+PORT=value
```

Comment:
Is this used anywhere in the PR? If it is not please remove it and add it in the PR that adds the `dotenv` package or something similar.

Reply:
In the future mini PR-s will be used because I want to learn and improve that skill. Now when I have some teamwork experience I can see why that approach is so better, important and preferable. I hope that is OK with you, since you will review them. 😁

Reply:
My general rule is not to have things added to the code/repo that will not be used in some way in the main branch after merging. Fewer things to review and keep track of. Even if you know it will be used it would be best to leave them to the PR that will actually use them.

I am very OK with mini PRs 😄